src/Sarif.WorkItems/WorkItemsResources.resx

```diff
@@ -117,28 +117,25 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="AdoSarifAddInMessage" xml:space="preserve">
-    <value>&lt;li&gt;Load the attached log file in the &lt;a href=""https://marketplace.visualstudio.com/_apis/public/gallery/publishers/WDGIS/vsextensions/MicrosoftSarifViewer/2.1.7/vspackage""&gt;Visual Studio SARIF add-in.&lt;/a&gt;&lt;/li&gt;</value>
+  <data name="AzureDevOpsDefaultDescriptionFooter" xml:space="preserve">
+    <value>&lt;br&gt;&lt;br&gt;For another viewing option, load the attached SARIF log file in: &lt;br&gt;
+&lt;li&gt;The &lt;a href=""https://marketplace.visualstudio.com/_apis/public/gallery/publishers/WDGIS/vsextensions/MicrosoftSarifViewer/2.1.7/vspackage""&gt;Visual Studio SARIF add-in&lt;/a&gt;.&lt;/li&gt;
```

Comment:
>"" [](start = 25, length = 2)

These should be just one quote. It's why the links don't function in the work items.